backends/ebpf/runtime/ebpf_runtime.c

```diff
@@ -94,51 +86,81 @@ static void write_pkts_to_pcaps(const char *pcap_base, pcap_list_array_t *output
     uint16_t arr_len = get_list_array_length(output_array);
     for (uint16_t i = 0; i < arr_len; i++) {
         char pcap_out_name[FILE_NAME_MAX];
-        snprintf(pcap_out_name, FILE_NAME_MAX, "%s%d%s", pcap_base, i, PCAPOUT);
+        if (snprintf(pcap_out_name, FILE_NAME_MAX, "%s%d%s", pcap_base, i, PCAPOUT) >=FILE_NAME_MAX) {
+            fprintf(stderr, "Filename %s%d%s too long. Truncated.\n"
```

Comment:
There is no need to say "truncated" since you terminate the program anyway. Just say that the filename is too long.

Reply:
But the best practice would be to actually allocate the filename buffer dynamically to be as big as needed. Using fixed-size buffers was done in the eighties when memories were too small.